### ISSUE TITLE
"Igoring" and "Following" badges should have custom translations

### ISSUE THREAD

**Issue by rob006** [clarkwinkelmann/flarum-ext-who-read#7]
Please do not reuse translations from `flarum-subscriptions`, since they're used in different contexts. `flarum-subscriptions` uses them always in first person context, while your extension uses them in third person context. Right now it is hard to translate this for some languages (like Polish).

https://github.com/clarkwinkelmann/flarum-ext-who-read/blob/afd3f2307e957ff3dd7d941ad2fecda36d83710d/js/src/forum/utils/appendReaderBadges.js#L9-L22

**Comment by rob006** [comment]
Also, supporting pluralization could be useful. Right now you have labels like "1 users read to the end".

**Comment by clarkwinkelmann** [comment]
Good call.

I have added keys for the subscription translations. By default they automatically re-use the value from core but language packs can now override them.

I have also changed the pluralized keys to take advantage of the pluralize feature of the translator. This caused a change in some keys.

I released the new version as 1.1.0.

I the process of adding the pluralized translations I found out the "explicit" rules were completely broken in Flarum. I made a patch for this extension and will push a PR to fix it in core.
